src/ServiceControl.Infrastructure.RavenDB/RavenHealthReporter.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using NServiceBus.Logging;
+using Raven.Client.Embedded;
+
+public class RavenHealthReporter
+{
+    public async void Start(EmbeddableDocumentStore documentStore, CancellationToken cancellationToken = default)
+    {
+        try
+        {
+            while (!cancellationToken.IsCancellationRequested)
+            {
+                var report = new StringBuilder();
+
+                var statistics = documentStore.DatabaseCommands.GetStatistics();
+
+                report.AppendLine("Report:");
+                report.AppendLine($"CountOfStaleIndexesExcludingDisabledAndAbandoned:  {statistics.CountOfStaleIndexesExcludingDisabledAndAbandoned}");
+
+                var indexes = statistics.Indexes.OrderBy(x => x.Name);
+
+                foreach (var indexStats in indexes)
+                {
+                    //var indexLag = unchecked((uint?)indexStats.IndexingLag);
```

Comment:
Remove?

Reply:
The `IndexLag` value sometimes is negative which I don't understand. How can lag be negative 😃 . The only thing I can image is that the underlying type is a uint so I asked it on the RavenDB forum https://groups.google.com/g/ravendb/c/wKyQqjjWHvc but no response yet.

Reply:
https://github.com/ravendb/ravendb/blob/2623ab17c207928110930fe481f78a35a5b0a01a/Raven.Abstractions/Data/IndexStats.cs

```c#
 public void SetLastDocumentEtag(Etag lastDocEtag)
{
    if (lastDocEtag == null)
        return;

    IndexingLag = (int) (lastDocEtag.Changes - LastIndexedEtag.Changes);

    if (lastDocEtag.Restarts != LastIndexedEtag.Restarts)
    {
        IndexingLag *= -1;
    }
}
```

Not sure what a negative value means but it is NOT unsigned. Going to make it absolute instead.